### DISCUSSION TITLE
Terraform plan error Kubernetes cluster unreachable try setting KUBERNETES_MASTER

### DISCUSSION THREAD

**Discussion by umar-shabbir** [eclipse-theia/theia-cloud#273] [Q&A]
I am trying to follow getting started for Minikube. Terraform plan worked fine once but now suddenly it is giving an error. I have tried deleting minikube and cleaning all terraform `.lock` files. Now after running `terraform plan` I am getting following error:
```

Changes to Outputs:
  ~ try_now = "https://192.168.59.109.nip.io/trynow/" -> (known after apply)

│ Error: Kubernetes cluster unreachable: invalid configuration: no configuration has been provided, try setting KUBERNETES_MASTER environment variable

│   with module.helm.helm_release.cert-manager,
│   on ..\..\modules\helm\main.tf line 93, in resource "helm_release" "cert-manager":
│   93: resource "helm_release" "cert-manager" {
│

│ Error: Kubernetes cluster unreachable: invalid configuration: no configuration has been provided, try setting KUBERNETES_MASTER environment variable

│   with module.helm.helm_release.theia-cloud-crds[0],
│   on ..\..\modules\helm\main.tf line 143, in resource "helm_release" "theia-cloud-crds":
│  143: resource "helm_release" "theia-cloud-crds" {


```
Does anyone know what could be reason for this?

Environment variable `KUBECONFIG` is set and points to `.kube` directory in my Windows user profile. This `.kube` directory contains the `yaml` file for minikube cluster

**Additional Information:**
Platform: Windows 11
Terraform: 1.7.0
Minikube: 1.32.0
VirtualBox: 7.0.12

**Comment by jfaltermeier** [comment]
Hi, I've found this terraform related issue, which reads a bit similar: https://github.com/terraform-aws-modules/terraform-aws-eks/issues/1234#issuecomment-787936210
Does setting `KUBE_CONFIG_PATH` help?

**Reply by umar-shabbir** [reply]
I also checked that issue. I had already set `KUBECONFIG`. Also tried with `KUBE_CONFIG_PATH` but still the same error appears
